miso-web/src/main/webapp/scripts/hot.js

```diff
@@ -515,13 +515,6 @@ var HotUtils = {
                         anyInvalidCells = false;
                         table
                             .validateCells(function() {
-                              if (anyInvalidCells) {
-                                failed.push('Please fix highlighted cells.');
-                                renderErrors();
-                                setSaveDisabled(false);
-                                ajaxLoader.classList.add('hidden');
-                                return;
-                              }
                               for (var i = 0; i < data.length; i++) {
                                 var errorHandler = function(errorMessage) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define this function outside of a loop. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AFunctionDefinitionInsideLoop)